sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -1067,11 +1064,11 @@ static ipv6_addr_t *_src_addr_selection(gnrc_netif_t *netif,
         uint8_t candidate_scope = _get_scope(ptr);
```

Comment:
:woman_shrugging: It doesn't even do that in the first place. There is just this comment saying "both link local", but it does not say what "both" is referring to and if this is an assumption or something else. This comment goes back to the [original implementation of this function](https://github.com/RIOT-OS/RIOT/pull/3561/files#diff-db9a1a910efb231b7ee7971f29055350R603) and even there it doesn't to anything with link local etc. It is not in the code I touched so I deem it unrelated, but if you want, I can add a piggy-back commit to remove it.

Reply:
> It is not in the code I touched so I deem it unrelated, but if you want, I can add a piggy-back commit to remove it.

Yes it is unrelated but since if was under rule 2, It got me confused. You can add the commit or not, as you said it is unrelated.